.styleci.yml

```diff
@@ -0,0 +1,18 @@
+preset: symfony
+
+enabled:
+  - align_double_arrow
+  - newline_after_open_tag
+  - ordered_use
+  - long_array_syntax
+  - php_unit_construct
+  - php_unit_strict
+
+disabled:
+  - psr0
```

Comment:
We are planning on making the web interface more and more friendly. One idea I've been toying with is allowing people to see the config used for each analysis on the each analysis page.